internal/repositories/equipment_test.go

```diff
@@ -710,3 +710,49 @@ func mapContainsEquipment(eq *ent.Equipment, m map[int]*ent.Equipment) bool {
 	}
 	return false
 }
+
+func IsTimeEqualWithHysteresis(t1 time.Time, t2 time.Time, hyst time.Duration) bool {
+	diff := t1.Sub(t2).Abs()
+	return diff <= hyst
+}
+
+func TestIsTimeEqualWithHysteresis(t *testing.T) {
+	// }
+	data := []struct {
+		t1       string
+		t2       string
+		hyst     string
+		layout   string
+		expected bool
+	}{
+		{"2006-01-02", "2006-01-03", "23h", time.DateOnly, false},
```

Comment:
time.DateOnly was added in go 1.20. The project is on go 1.19.

Reply:
May be it's time to update the go version? And ent as well, because there incompatibility of specific ent version and go 1.22 as we discussed before in the Teams chat https://github.com/ent/ent/issues/3864 and we need to keep both go and ent up to date?

Reply:
I removed cases using time.DateOnly. Can restore them after we update from 1.19